theories/_CoqProject

```diff
@@ -9,37 +9,38 @@
 # ./Algorithmic/Typing/Definitions.v
 # ./Algorithmic/Typing/Lemmas.v
 ./Core/Base.v
-# ./Core/Completeness.v
-# ./Core/Completeness/Consequences/Rules.v
-# ./Core/Completeness/Consequences/Types.v
-# ./Core/Completeness/ContextCases.v
-# ./Core/Completeness/FunctionCases.v
-# ./Core/Completeness/FundamentalTheorem.v
-# ./Core/Completeness/LogicalRelation.v
-# ./Core/Completeness/LogicalRelation/Definitions.v
-# ./Core/Completeness/LogicalRelation/Lemmas.v
-# ./Core/Completeness/LogicalRelation/Tactics.v
-# ./Core/Completeness/NatCases.v
-# ./Core/Completeness/SubstitutionCases.v
-# ./Core/Completeness/SubtypingCases.v
-# ./Core/Completeness/TermStructureCases.v
-# ./Core/Completeness/UniverseCases.v
-# ./Core/Completeness/VariableCases.v
+./Core/Completeness.v
+./Core/Completeness/Consequences/Rules.v
+./Core/Completeness/Consequences/Types.v
+./Core/Completeness/ContextCases.v
+./Core/Completeness/EqualityCases.v
+./Core/Completeness/FunctionCases.v
+./Core/Completeness/FundamentalTheorem.v
+./Core/Completeness/LogicalRelation.v
+./Core/Completeness/LogicalRelation/Definitions.v
+./Core/Completeness/LogicalRelation/Lemmas.v
+./Core/Completeness/LogicalRelation/Tactics.v
+./Core/Completeness/NatCases.v
+./Core/Completeness/SubstitutionCases.v
+./Core/Completeness/SubtypingCases.v
+./Core/Completeness/TermStructureCases.v
+./Core/Completeness/UniverseCases.v
+./Core/Completeness/VariableCases.v
 # ./Core/Semantic/Consequences.v
-# ./Core/Semantic/Domain.v
-# ./Core/Semantic/Evaluation.v
-# ./Core/Semantic/Evaluation/Definitions.v
-# ./Core/Semantic/Evaluation/Lemmas.v
-# ./Core/Semantic/Evaluation/Tactics.v
-# ./Core/Semantic/NbE.v
-# ./Core/Semantic/PER.v
-# ./Core/Semantic/PER/CoreTactics.v
-# ./Core/Semantic/PER/Definitions.v
-# ./Core/Semantic/PER/Lemmas.v
-# ./Core/Semantic/Readback.v
-# ./Core/Semantic/Readback/Definitions.v
-# ./Core/Semantic/Readback/Lemmas.v
-# ./Core/Semantic/Realizability.v
+./Core/Semantic/Domain.v
```

Comment:
I would say the files from here and below are natural to go under a smaller PR.

Reply:
It's basically the first commit of this PR.